### ISSUE TITLE
Front matter `permalink` seems to mess up

### ISSUE THREAD

**Issue by nc7s** [saneef/eleventy-plugin-asciidoc#3]
Using eleventy 1.0.1 and this plugin 1.3.1.

Adding `permalink` in the front matter:

```asciidoc
<!-- hello.adoc -->
---
permalink: /hello-permalink
---

= Hello Title

Hello content
```

Running eleventy:

```
❯ yarn run eleventy
yarn run v1.22.18
warning ../../package.json: No license field
$ /Users/i/Works/mstest/node_modules/.bin/eleventy
[11ty] Writing _site/<div class="paragraph">
[11ty] <p>Hello content</p>
[11ty] </div> from ./hello.adoc
```

which seems the rendered content is being used as permalink. I dug into eleventy source code a little, but have yet to find the crux of this.

**Comment by saneef** [comment]
@bnoctis thanks for pointing out this.

I have published a new version of the plugin `v1.3.2`. Could you try with that?

Also, don't forget [the trailing slash in permalink](https://www.11ty.dev/docs/permalinks/#remapping-output-(permalink)). If you want to generate file at `/hello-permalink/index.html`, you need to set the permalink to either `/hello-permalink/` or `/hello-permalink/index.html`.

**Comment by nc7s** [comment]
Solved! Thanks for such a quick fix.

I was actually trying to make posts have *no* trailing slash, for the sake of directory and file semantics.
For now, I'm using `permalink: /posts/something.html`. Thinking of a plugin or something that enables this for all non-`index` things.
